### DISCUSSION TITLE
How to have a simple textarea ?

### DISCUSSION THREAD

**Discussion by hotgeart** [Lakshan-Madushanka/laravel-comments#25] [General]
Hi,

I'm testing this package, and I like it, but I'm not a fan of giving my users the options for headers, lists, etc. I don't see any configuration for excluding Quill.js. I tried removing the toolbar from Quill.js, but it's buggy. Here's a video showing the issue: https://www.youtube.com/watch?v=U2RWCy4oEEM

How can I implement a simple textarea/remove Quill.js? Thanks!

**Comment by hotgeart** [comment]
Ok I found it

\resources\views\vendor\comments\livewire\editor.blade.php

    <div>
        <div wire:ignore>
            <textarea name="comment" wire:model.defer="text" class="min-h-32 rounded rounded-t-none pointer-events"></textarea>
                <livewire:comments-user-list :$guestModeEnabled :$editorId />
            </div>
        </div>
    </div>

**Reply by Lakshan-Madushanka** [reply]
You also have to change the js in the bottom to make it work other functionalities like mentioning other users, or you can simple change quill config here https://github.com/Lakshan-Madushanka/laravel-comments/blob/cd9d170e8f7ee6bb977c9c18de43810e956ce57c/config/comments.php#L155

I think I haven't  mention it in documentation guessing it can be easily found  in config file. It might be good to add it in documentation.

[Here](https://github.com/Lakshan-Madushanka/laravel-comments-documentation) is the repo of the documentation. Proper pull requests are more than welcome  🙏.

**Reply by Lakshan-Madushanka** [reply]
Sorry, I made this answer without watching your video that showing editor is disabled all the time. I will test and update the answer.

**Reply by Lakshan-Madushanka** [reply]
That's was a little bug and move to here #26

**Comment by Lakshan-Madushanka** [comment]
This was fixed for v1 with the release  [v1.4.5](https://github.com/Lakshan-Madushanka/laravel-comments/releases/tag/1.4.5)
